VFS Tests failing in JDK 17

There are multiple tests failing when running with JDK 17. 

<pre>
[ERROR]   AccumuloReloadingVFSClassLoaderTest.testReloading:106 » ClassNotFound test.HelloWorld
[ERROR]   AccumuloReloadingVFSClassLoaderTest.testReloadingWithLongerTimeout:152 » ClassNotFound test.HelloWorld
[ERROR]   AccumuloVFSClassLoaderTest.testDefaultContextConfigured:139 » ClassNotFound test.HelloWorld
[ERROR]   ContextManagerTest.differentContexts:116 » ClassNotFound test.HelloWorld
[ERROR]   ContextManagerTest.testPostDelegation:133 » ClassNotFound test.HelloWorld
[ERROR]   VfsClassLoaderTest.testGetClass:66 » ClassNotFound test.HelloWorld
</pre>

I opened a ticket with Commons VFS: https://issues.apache.org/jira/browse/VFS-819

--- a/start/src/main/java/org/apache/accumulo/start/classloader/vfs/AccumuloVFSClassLoader.java
+++ b/start/src/main/java/org/apache/accumulo/start/classloader/vfs/AccumuloVFSClassLoader.java
@@ -285,6 +285,7 @@ public class AccumuloVFSClassLoader {
     vfs.addExtensionMap("tbz2", "tar");
     vfs.addExtensionMap("tgz", "tar");
     vfs.addExtensionMap("bz2", "bz2");
+    vfs.addMimeTypeMap("application/java-archive", "jar");
     vfs.addMimeTypeMap("application/x-tar", "tar");
     vfs.addMimeTypeMap("application/x-gzip", "gz");
     vfs.addMimeTypeMap("application/zip", "zip");
@@ -426,9 +427,7 @@ public class AccumuloVFSClassLoader {
   private static synchronized ContextManager getContextManager() throws IOException {
     if (contextManager == null) {
       getClassLoader();
-      contextManager = new ContextManager(generateVfs(), () -> {
-        return getClassLoader();
-      });
+      contextManager = new ContextManager(generateVfs(), AccumuloVFSClassLoader::getClassLoader);
     }
 
     return contextManager;

--- a/start/src/test/java/org/apache/accumulo/start/test/AccumuloDFSBase.java
+++ b/start/src/test/java/org/apache/accumulo/start/test/AccumuloDFSBase.java
@@ -112,6 +112,7 @@ public class AccumuloDFSBase {
       vfs.addExtensionMap("tbz2", "tar");
       vfs.addExtensionMap("tgz", "tar");
       vfs.addExtensionMap("bz2", "bz2");
+      vfs.addMimeTypeMap("application/java-archive", "jar");
       vfs.addMimeTypeMap("application/x-tar", "tar");
       vfs.addMimeTypeMap("application/x-gzip", "gz");
       vfs.addMimeTypeMap("application/zip", "zip");

